### ISSUE TITLE
About quick start on 4k start

### ISSUE THREAD

**Issue by MeLonJ10** [Tengfei-Wang/DCSR#4]
Thanks for sharing the great work.
I use the following command to evaluation on camera fusion.
"python3 main.py --pre_train  './experiments/pretrain/model_basic.pt'  --save test_4k  --test_only  --n_feats 64"
However, the displayed result ([CameraFusion x2]	PSNR: 34.490) is quite different from the value reported in the paper ( on table 2, psnr=36.98 when using ours-l1 and psnr=34.41 when using ours).
Can you offer me some help?
Thanks a lot!

**Comment by Tengfei-Wang** [comment]
Hi,
As we changed some variable and function names in the code, the released checkpoint file is a retrained version. Also, the released model (PSNR:34.49) is actually slightly better than the reported one (PSNR:34.41) in terms of PSNR. To produce ours-l1 (PSNR: 36.98) in table 2, you need to train the model with L1 loss only (i.e., disable other losses). As observed in many SR works, models trained with L1 or L2 loss only usually perform better quantitatively but suffer inferior visual quality.

**Comment by MeLonJ10** [comment]
I see, thanks for the reply~
